android/src/test/java/io/keen/client/android/AndroidJsonHandlerTest.java

```diff
@@ -52,12 +68,17 @@ public void writeSimpleMap() throws Exception {
         map.put("boolean_property", true);
 
         // Serialize the map.
-        String result = serialize(map);
+        serialize(map);
```

Comment:
Correct; the mocked `JsonObjectManager` will return `null` from `newObject` and `newArray`, so the result ends up being meaningless. We're just verifying that it calls things the way we expect it to.